src/routes/controllers/courses.controller.ts

```diff
@@ -35,41 +38,35 @@ export const addCourse = async (req: Request | any, res: Response) => {
   try {
     // manc la validazione del body
     const { name, status, year } = req.body;
-    const result: Courses = await add(name, status, year, req.userData);
-    return res.status(201).json(result);
+    const course: Courses = await add(name, status, year, req.userData);
+    return res.status(201).json(mapCourseData(course));
```

Comment:
La post per aggiungere un nuovo corso mi da sempre codice 401 anche se ho tolto le funzioni `isAuth` e `isAdmin`. Nel messaggio di errore non ritorna nessun errore.

Reply:
io riesco a creare perfettamente un nuovo corso:
`{
    "id": "5e37fe43d3a7ed72a5a5038a",
    "name": "testing-10",
    "status": "pedding",
    "year": "Wed Jan 01 2020",
    "students": [],
    "teachers": [],
    "subjects": []
}`

potresti avere una versione in locale differente dalla mia, fai un `git stash` e un `git pull` per assicurarti di avere l'ultima versione.

Reply:
Ok, non avevo una versione aggiornata. Perfetto per me può andare